### DISCUSSION TITLE
Some issues I encountered while using Phyrox

### DISCUSSION THREAD

**Discussion by TTKings** [community/community#115020] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

"I always set the language to Simplified Chinese, but every time I restart the browser, it changes back to English. When I install plugins and then restart, the plugins are lost. I cannot switch my account to China when logging in. Also, there are synchronization issues when logging in with a proxy. Please help me improve. a user from China."

**Comment by queenofcorgis** [comment]
If you're asking how to give feedback on software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
